### PR TITLE
Update Japanese style properties documentation

### LINKED ISSUE #2556: Props provided by Yamada UI such as `lineClamp` are not listed
### Provide the exact quote of the error or issue

Props provided by Yamada UI such as `lineClamp` are not listed.

### Does the docs page already exist? Please link to it.

https://yamada-ui.com/styled-system/style-props

### Are you willing to participate in fixing this issue and create a pull request with the fix?

Yes

### PR DESCRIPTION
**Pull request by hirotomoyamada**
Closes #2556

## Description

Update Japanese style properties documentation.

## Is this a breaking change (Yes/No):

No

### CODE REVIEW

File: docs/scripts/content/styled-system.ts [hunk 1]
```diff
@@ -127,14 +132,17 @@ const getJSDocs = (node: TypeAliasDeclaration) => (sourceFile: SourceFile) => {
 
     if (!hasJSDoc(member)) return
 
-    member.jsDoc.forEach(({ tags }) =>
+    member.jsDoc.forEach(({ tags, comment }) => {
+      data.description =
+        typeof comment === "string" ? comment : comment?.join("\n")
```
Comment (taroj1205) [inline docs/scripts/content/styled-system.ts:137]: Shouldn't we use `isString` from @yamada-ui/utils?

File: scripts/generate-css/styles.ts [hunk 2]
```diff
@@ -154,90 +159,85 @@ export const generateStyles = async (
...
-      })(true)
+    ]: [string, UIOptions],
+    targetStyles: string[],
+  ) => {
+    if (isProcessSkip) processSkipProperties.push(prop)
+
+    const relatedStyles = styles.filter(({ prop }) =>
+      typeof properties === "string"
+        ? prop === properties
+        : properties?.includes(prop),
```
Comment (taroj1205) [inline scripts/generate-css/styles.ts:197]: Shouldn't we use `isString` provided by @yamada-ui/utils?

File: docs/scripts/content/styled-system.ts (hunk 1 above)
Comment (hirotomoyamada) [reply docs/scripts/content/styled-system.ts:137]: @taroj1205 

You cannot use `@yamada-ui/utils` or `@yamada-ui/react` here.

File: scripts/generate-css/styles.ts (hunk 2 above)
Comment (hirotomoyamada) [reply scripts/generate-css/styles.ts:197]: @taroj1205 

You cannot use `@yamada-ui/utils` or `@yamada-ui/react` here.

**Review by kurakke — APPROVED** [review]

**Review by taroj1205 — APPROVED** [review]
LGTM

**Pull request merged**